build/root/WORKSPACE

```diff
@@ -66,3 +50,10 @@ docker_pull(
     registry = "gcr.io",
     repository = "google-containers/debian-iptables-amd64",
 )
+
+docker_pull(
+    name = "official_busybox",
+    registry = "index.docker.io",
+    repository = "library/busybox",
+    tag = "latest",  # TODO: more hermetic?
```

Comment:
per discussion in https://groups.google.com/forum/#!topic/bazel-discuss/CX54hxCb6e8, the only correct way to do this is to use the digest; otherwise we'll end up with the problem we had before we started using `--pull` everywhere.

remembering to update the digest is going to be a pain, though.